### PR TITLE
Validate attributes based on their types

### PR DESCRIPTION
**Pull request by orhoj**
## Purpose
Validate that add credential requests do not contain invalid attributes.

## Changes
- Added validation when receiving a request to add a credential.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: packages/browser-wallet/src/background/web3Id.ts [hunk 1]
```diff
@@ -126,28 +140,40 @@ export const runIfValidWeb3IdCredentialRequest: RunCondition<MessageStatusWrappe
...
-                    success: false,
-                    message: `Credential issuer network is not the same as the current wallet network`,
-                },
-            };
+            return rejectRequest(`Credential issuer network is not the same as the current wallet network`);
         }
+
+        for (const attributeEntry of Object.entries(credential.credentialSubject.attributes)) {
+            const attributeKey = attributeEntry[0];
+            const attributeValue = attributeEntry[1];
```
Comment (shjortConcordium) [inline packages/browser-wallet/src/background/web3Id.ts:153]:
```suggestion
        for (const [attributeKey, attributeValue] of Object.entries(credential.credentialSubject.attributes)) {
```
:sunglasses:

File: packages/browser-wallet/src/background/web3Id.ts [hunk 2]
```diff
@@ -35,13 +35,20 @@ import {
...
 import { WAIT_FOR_CLOSED_POPUP_ITERATIONS, WAIT_FOR_CLOSED_POPUP_TIMEOUT_MS } from '@shared/constants/web3id';
+import { Buffer } from 'buffer/';
 import { openWindow, RunCondition, testPopupOpen } from './window-management';
 import bgMessageHandler from './message-handler';
 
 const NO_CREDENTIALS_FIT = 'No temporary credentials fit the given id';
 const INVALID_CREDENTIAL_PROOF = 'Invalid credential proof given';
 const MISSING_CREDENTIAL_PROOF = 'No credential proof given';
 
+const MAX_U64 = 2n ** 64n;
```
Comment (shjortConcordium) [inline packages/browser-wallet/src/background/web3Id.ts:46]:
```suggestion
const MAX_U64 = 2n ** 64n - 1;
```
:thinking:

**Review by shjortConcordium — APPROVED** [review]

File: packages/browser-wallet/src/background/web3Id.ts (hunk 2 above)
Comment (orhoj) [reply packages/browser-wallet/src/background/web3Id.ts:46]: Yes, lol.

File: packages/browser-wallet/src/background/web3Id.ts (hunk 1 above)
Comment (orhoj) [reply packages/browser-wallet/src/background/web3Id.ts:153]: Nice. I will do that.

**Pull request merged**